openmetadata-ui/src/main/resources/ui/src/rest/tagAPI.ts

```diff
@@ -108,6 +108,17 @@ export const patchClassification = async (id: string, data: Operation[]) => {
   return response.data;
 };
 
+export const getTagByName = async (
+  name: string,
```

Comment:
As we are passing the FQN, so function name should be `getTagByFqn` also the argument should be `fqn` instead of the name.
```suggestion
export const getTagByFqn = async (
  fqn: string,
```